interfaces/mutual_tls/v0/README.md

```diff
@@ -0,0 +1,51 @@
+# `mutual_tls`
+
+## Usage
+
+Some charms require knowledge of other charms public TLS certificates for trusted communication with them. This relation interface describes the expected behaviour of any charm claiming to be able to authenticate other charms through TLS communication.
+
+**Warning**: This relation interface should not be used by charms requiring certificates for themselves. Please use the `tls-certificates` interface for this usecase.
+
+## Direction
+
+```mermaid
+flowchart TD
+    Provider -- certificate, ca, chain --> Requirer
+```
+
+As with all Juju relations, the `mutual_tls` interface consists of two parties: a Provider and a Requirer.
+
+## Behavior
+
+Both the Requirer and the Provider need to adhere to criteria to be considered compatible with the interface.
+
+### Provider
+
+- Is expected to provide its own public certificate, its CA certificate and its CA chain.
+
+### Requirer
+
+- Is expected to use the provider's certificate and chain to authenticate the provider when communicating with it.
```

Comment:
**question:** Do we want to make all of `ca/certificate/chain` optional? 

I don't think it's necessary to establish a mutual TLS connection to have all 3, and I can imagine scenarios where we might not want to share one or the other, or might not have it.

Reply:
Right now, only the certificate is marked as "required" and the ca and chain are optional.I can change it for all 3 fields to be optional.

Reply:
Done